mono/mini/mini-runtime.h

```diff
@@ -457,12 +457,14 @@ gboolean          mono_running_on_valgrind (void);
 
 MonoLMF * mono_get_lmf                      (void);
 #define mono_get_lmf_addr mono_tls_get_lmf_addr
-MonoLMF** mono_get_lmf_addr                 (void);
+//#include "mono/utils/mono-tls-inline.h" for this.
+//MonoLMF** mono_get_lmf_addr                 (void);
```

Comment:
It has been so long I don't remember, but since it commented out, I guess removed.

Reply:
So remove then? Should we also nuke the #define mono_get_lmf_addr mono_tls_get_lmf_addr since that is not complete either unless you include mono-tls-inline.h?

Reply:
I removed the two commented out lines. And rebased to master.

Reply:
I also moved mono_get_lmf_addr from mini-runtime.h to mono-tls-inline.h.
Mostly cosmetic, but reasonable.
It was "better" before, for the case of token pasting or stringizatoin, but should be equivalent either way.